plugins/tech-insights/src/api/TechInsightsClient.ts

```diff
@@ -29,16 +29,14 @@ import {
   defaultCheckResultRenderers,
 } from '../components/CheckResultRenderer';
 
-export type Options = {
-  discoveryApi: DiscoveryApi;
-  identityApi: IdentityApi;
-};
```

Comment:
This introduced a warning in `api-report.md` because it was not exported by the plugin. I think it is common to just inline this type instead of it being explicit.

Reply:
Yep, that or `TechInsightsClientOptions`, think either is fine